### PR TITLE
Computer Vendor Redux

### PR DESCRIPTION
**Pull request by Geevies**
* The computer vendor can now vend out Laptops, Tablets, and Wristbound computers.
* The computer vendor has more things that can be vended out.
* You can now pay for the computer vendor with an ID or a PDA or modular computer containing an ID.
* You can now use the AI maintenance program on laptops too.
* Hardware in the modular computer configuration program now have capital letters.

### CODE REVIEW

**Comment by Geevies** [comment]
!review

File: code/modules/modular_computers/laptop_vendor.dm [hunk 1]
```diff
@@ -261,103 +254,109 @@
...
+			user.visible_message(SPAN_NOTICE("\The [user] begins securing \the [src] to the floor."), SPAN_NOTICE("You start securing \the [src] to the floor."))
+		if(do_after(user, 20 / W.toolspeed))
 			if(!src)
 				return
-			to_chat(user, span("notice", "You [anchored ? "un" : ""]secured \the [src]!"))
+			to_chat(user, SPAN_NOTICE("You [anchored ? "un" : ""]secured \the [src]!"))
 			anchored = !anchored
 		return
+	else if(state != 2 && (istype(W, /obj/item/card/id) || istype(W, /obj/item/device/pda) || istype(W, /obj/item/modular_computer) || istype(W, /obj/item/card/tech_support)))
+		to_chat(user, SPAN_WARNING("\The [src] isn't in the payment mode yet!"))
```
Comment (NonQueueingMatt) [inline code/modules/modular_computers/laptop_vendor.dm:274]: Bit too gamey. Could just tell them that they're having trouble slamming their ID into a slot that's not open :^)

Comment (Geevies) [inline code/modules/modular_computers/laptop_vendor.dm:274]:
```suggestion
		to_chat(user, SPAN_WARNING("\The [src]'s ID Scan slot isn't open yet!"))
```

**Comment by Geevies** [comment]
ha ha vueui

**Pull request closed without merging**